fastplotlib/graphics/linecollection.py

```diff
@@ -1,16 +1,20 @@
 import numpy as np
 import pygfx
-from typing import Union
+from typing import Union, List
+
+from fastplotlib.graphics._base import Graphic
+
 from .line import LineGraphic
 from typing import *
+from ._base import Interaction
 
 
-class LineCollection():
-    def __init__(self, data: List[np.ndarray], z_position: Union[List[float], float] = None, size: Union[float, List[float]] = 2.0, colors: Union[List[np.ndarray], np.ndarray] = None,
+class LineCollection(Interaction):
+    def __init__(self, data: List[np.ndarray], zlevel: Union[List[float], float] = None, size: Union[float, List[float]] = 2.0, colors: Union[List[np.ndarray], np.ndarray] = None,
                  cmap: Union[List[str], str] = None, *args, **kwargs):
 
-        if not isinstance(z_position, float) and z_position is not None:
-            if not len(data) == len(z_position):
+        if not isinstance(zlevel, float) and zlevel is not None:
```

Comment:
same as above, I renamed this to `z_position`

Reply:
oops, was trying to fix stupid merge conflicts

should be fixed now